curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6746005e0cf96b701d861793.md

```diff
@@ -0,0 +1,108 @@
+---
+id: 6746005e0cf96b701d861793
+title: Task 29
+challengeType: 19
+dashedName: task-29
+---
+
+<!-- (Audio) Lisa: Wasn't it stressful though? -->
+
+# --description--
+
+Listen to the audio and answer the question.
+
+# --questions--
+
+## --text--
+
+What is Lisa asking James about?
+
+## --answers--
+
+She's asking if it took too long to find the bug.
+
+### --feedback--
+
+Lisa is not asking about how long it took. She just wants to know if it was stressful.
+
+---
+
+She's trying to confirm if finding the bug was a stressful task.
+
+---
+
+She's asking if James was the one who didn't update the credentials.
+
+### --feedback--
+
+Lisa is not trying to know who's responsible for it.
+
+---
+
+She's asking if there is anything she can do to help.
+
+### --feedback--
+
+Help is no longer necessary, since James has already fixed the problem.
+
+## --video-solution--
+
+2
+
+# --explanation--
+
+The word `stressful` describes situations or events that cause stress, tension, or anxiety. It is often used to talk about experiences that are challenging or demanding. For example:
+
+- `The deadline was stressful because we had so much work to do.` (meaning they felt a lot of stress due to the deadline)
+
+- `Presenting to a large audience can be stressful.` (meaning you can be feel really anxious when you present in front of an audience)
```

Comment:
```suggestion
- `The deadline was stressful because we had so much work to do.` - Meaning they felt a lot of stress due to the deadline.

- `Presenting to a large audience can be stressful.` - Meaning you can be feel really anxious when you present in front of an audience.
```